test/spec/EnrollU2F_spec.js

```diff
@@ -0,0 +1,156 @@
+/*jshint maxparams:11 */
+define([
+  'okta',
+  'jquery',
+  '@okta/okta-auth-js/jquery',
+  'helpers/mocks/Util',
+  'helpers/dom/EnrollU2FForm',
+  'helpers/dom/Beacon',
+  'helpers/util/Expect',
+  'sandbox',
+  'LoginRouter',
+  'helpers/xhr/MFA_ENROLL_allFactors',
+  'helpers/xhr/MFA_ENROLL_ACTIVATE_U2F'
+],
+function (Okta,
+          $,
+          OktaAuth,
+          Util,
+          Form,
+          Beacon,
+          Expect,
+          $sandbox,
+          Router,
+          resAllFactors,
+          resEnrollActivateU2F) {
+
+  var itp = Expect.itp;
+  var tick = Expect.tick;
+
+  Expect.describe('EnrollSecurityKey(U2F)', function () {
+
+    function setup(startRouter) {
+      var setNextResponse = Util.mockAjax();
+      var baseUrl = 'https://foo.com';
+      var authClient = new OktaAuth({url: baseUrl});
+      var router = new Router({
+        el: $sandbox,
+        baseUrl: baseUrl,
+        authClient: authClient,
+        globalSuccessFn: function () {}
+      });
+      Util.mockRouterNavigate(router, startRouter);
+      return tick()
+      .then(function () {
+        setNextResponse(resAllFactors);
+        router.refreshAuthState('dummy-token');
+        return tick();
+      })
+      .then(function () {
+        router.enrollU2F();
+        return tick();
+      })
+      .then(function () {
+        return {
+          router: router,
+          beacon: new Beacon($sandbox),
+          form: new Form($sandbox),
+          ac: authClient,
+          setNextResponse: setNextResponse
+        };
+      });
+    }
+
+    function mocku2fEnroll() {
+      window.u2f = {
```

Comment:
I tried that, but window.u2f is undefined. I tried including the 'u2f-api-polyfill' in this test, but that still didn't work. So I stayed with the overriding approach.